Info.plist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this file to be at the root?

Reply:
No, I can move it up to the ffi/dotnet

Reply:
If it’s dotnet specific, I would prefer. Thank you!